docs/pywbemclicmdshelp.rst

```diff
@@ -686,23 +686,26 @@ The following defines the help output for the `pywbemcli connection add --help`
 
 ::
 
-    Usage: pywbemcli connection add [COMMAND-OPTIONS]
+    Usage: pywbemcli connection add [COMMAND-OPTIONS] NAME
 
       Add a new WBEM connection definition from specified options.
 
       Create a new WBEM connection definition in the connections file from the
       specified options. A connection definition with that name must not yet
       exist.
 
-      The current connection remains unchanged by this command.
+      The NAME argument MUST exist. It define the server uri and the unique name
```

Comment:
To me it does not sound OK to talk about "existing" options. Suggest to change "The NAME argument MUST exist." to "The NAME argument is required.".